webapp/graphite/util.py

```diff
@@ -72,13 +72,20 @@ def getProfileByUsername(username):
   except Profile.DoesNotExist:
     return None
 
-
 def is_local_interface(host):
+  is_ipv6 = False
   if ':' in host:
-    host = host.split(':',1)[0]
+    try:
+      socket.inet_pton(socket.AF_INET6, host)
+      is_ipv6 = True
```

Comment:
Not really, if host is already a valid ipv6 address then we don't need to manipulate it further, if it's not then socket.error will be raised and we'll split the port out of it and then everything will proceed as it did before, should work unless there's something I'm missing.